### DISCUSSION TITLE
Wert Runden

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#1227] [Q&A]
Mit JSON Formatierung kenn ich mich nicht so aus. Immer nur erfolgreich Copy&Paste. ;-P

Wir bekomme ich eine Zahl mit JSON gerundet? Also von 1,23456789 auf 1,23?

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/beispiele/geraete-einstellungen/datenpunkt-eigenschaften/nachkommastellen-auto

**Comment by Zefau** [comment]
Bitte für Fragen die Discussions nutzen
